### PR TITLE
Changes for class PublicLawRestriction

### LINKED ISSUE #1178: Changes for class PublicLawRestriction
Different changes on attribute level and multilingual elements:
- information - attribute renamed in legend_text to homogenize it with the same content in LegendEntry
- other_theme - attribute removed
- symbol - the graphical representation of this plr
- [x] length_share - new attribute already implemented in pyramid_oereb
- [x] nbr_of_points - new attribute already implemented in pyramid_oereb

### What needs to be done:
- [x] rename 'information' attribute to 'legend_text' from model definition throughout the code to data-extract https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/lib/records/plr.py#L30
- [x] get rid of all other_theme occurences

### Standard model definitions for db (standard models are created through templates!):
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/standard/templates/plr_integer_primary_keys.py.mako#L306-L362
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/standard/templates/plr_string_primary_keys.py.mako#L306-L362

### PLR sources definition:
https://github.com/openoereb/pyramid_oereb/blob/c5c6cb571027b41bf158f3118ae3bfff4a60be6d/pyramid_oereb/standard/sources/plr.py

### Record definition:
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/lib/records/plr.py

### Import Script:
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/standard/xtf_import/public_law_restriction_document.py

### Renderer:
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/lib/renderer/extract/templates/xml/public_law_restriction.xml
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/lib/renderer/extract/json_.py#L239-L315

### Hook Methods:
https://github.com/openoereb/pyramid_oereb/blob/787ce93db13fa9af2b775b9a80a7b4078fee56a3/pyramid_oereb/standard/hook_methods.py#L123

### Tests:
https://github.com/openoereb/pyramid_oereb/blob/787ce93db13fa9af2b775b9a80a7b4078fee56a3/tests/records/test_plr.py
https://github.com/openoereb/pyramid_oereb/blob/787ce93db13fa9af2b775b9a80a7b4078fee56a3/tests/test_processor.py#L169
https://github.com/openoereb/pyramid_oereb/blob/787ce93db13fa9af2b775b9a80a7b4078fee56a3/tests/renderer/xml/test_public_law_restriction.py
https://github.com/openoereb/pyramid_oereb/blob/787ce93db13fa9af2b775b9a80a7b4078fee56a3/tests/renderer/test_json.py#L264

### PR DESCRIPTION
**Pull request by vuilleumierc**
Closes #1178

### CODE REVIEW

File: pyramid_oereb/lib/records/plr.py [hunk 1]
```diff
@@ -27,13 +27,13 @@ class PlrRecord(EmptyPlrRecord):
     Public law restriction record.
     """
 
-    def __init__(self, theme, information, law_status, published_from, responsible_office, symbol,
```
Comment (vvmruder) [inline pyramid_oereb/lib/records/plr.py:30]: type_code_list should be dictionary type for multilingual usage now

**Review by vvmruder — CHANGES_REQUESTED** [review]
As I understand the document, the type_code_list has to be of type multilingual now. So in Models this means JSONType and in Records this means Dictionary. In All other places handling should be adapted to this new type.

File: pyramid_oereb/lib/records/plr.py (hunk 1 above)
Comment (vuilleumierc) [reply pyramid_oereb/lib/records/plr.py:30]: Thank for your review - it was not clear to me but I will do this change now

**Review by marionb — APPROVED** [review]
Can you rebase to master?
other than that LGTM! thanks.

**Comment by vvmruder** [comment]
Yes, this looks ok. The 9 failing tests are expected ones.

It can be merged.

**Review by vvmruder — APPROVED** [review]

**Pull request merged**